### PR TITLE
[TEVA-3306] Create hiring staff landing page

### PR DESCRIPTION
**Pull request by josephhull676**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-3306

## Design

- https://www.figma.com/file/StE5Vg64YAZj7LrHidsoEG/Sprint-92?node-id=141%3A2185

## Questions

- Added a div with the class 'spacer' to add the gap at the bottom (as seen in the designs). Is there a better way of doing this?

### CODE REVIEW

File: app/frontend/src/styles/application.scss [hunk 1]
```diff
@@ -23,6 +23,7 @@ $govuk-new-link-styles: true;
 @import 'application/full-width-mast';
 @import 'application/govuk-notification-banner';
 @import 'application/govuk-phase-banner';
+@import 'application/hiring-staff-landing-page';
```
Comment (alexbowen) [inline app/frontend/src/styles/application.scss:26]: whilst the current 'structure' of the css folders/files probably has limited shelf life with plans i have - could this file go in the publishers (hiring staff) folder to keep consistency with the current status quo? and then the file name is obviously just `landing-page`

File: app/views/layouts/landing_page.html.slim [hunk 2]
```diff
@@ -0,0 +1,50 @@
+doctype html
```
Comment (alexbowen) [inline app/views/layouts/landing_page.html.slim:1]: does this really need its own unique layout file and opposed to using the default?

File: app/views/pages/list-school-job.html.slim [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+    .govuk-grid-row.section class="govuk-!-padding-left-3"
+      .govuk-grid-column-one-third class="govuk-!-padding-left-0"
+        h3.govuk-heading-m class="govuk-!-margin-top-4"
+          | Sign in
+
+        = govuk_link_to("Sign in and list your jobs for free", publishers_sign_in_path, class: "govuk-body-m govuk-!-margin-bottom-4")
+
+        = govuk_link_to("Request a Teaching Vacancies account", publishers_account_requests_new_path, class: "govuk-body-m govuk-!-margin-bottom-4")
+
+    .spacer
```
Comment (alexbowen) [inline app/views/pages/list-school-job.html.slim:229]: why not just a margin or padding bottom?

**Comment by alexbowen** [comment]
how do i see this page? just wanted to have a look on mobile/diff screen widths. Or if you have already and are happy could you pop a few screen shots in description if this PR?

**Comment by alexbowen** [comment]
shouldnt the word million sit alongside the number?

Screenshot 2021-11-29 at 18 37 51

**Comment by alexbowen** [comment]
mobile view 

Screenshot 2021-11-29 at 18 39 54

File: app/views/pages/list-school-job.html.slim [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+        .account-actions
+          = govuk_start_button(text: "Sign in", href: page_path("sign-in"))
+
+          .request-account-link
+            span.govuk-body-m class="govuk-!-margin-left-5"
+              = "or"
+            = govuk_link_to("request a Teaching Vacancies account", page_path("dsi-account-request"), class: "govuk-body-m govuk-!-margin-left-1")
+
+      .govuk-grid-column-one-third class="govuk-!-padding-left-0"
+        img class="govuk-!-margin-top-4" src=asset_pack_path("media/images/hiring_staff_landing_page/header_icons.png")
```
Comment (alexbowen) [inline app/views/pages/list-school-job.html.slim:29]: the images should have dimensions (and alt tags @david-mears-dfe already mentioned) else the browser is going display them at original size and this is partly why you are going to have issues on mobile - see screen shot below

File: app/views/layouts/landing_page.html.slim (hunk 2 above)
Comment (josephhull676) [reply app/views/layouts/landing_page.html.slim:1]: I didn't know how else to not include things such the links in the header

File: app/views/pages/list-school-job.html.slim (hunk 3 above)
Comment (josephhull676) [reply app/views/pages/list-school-job.html.slim:229]: The largest spacing override is `govuk-!-margin/padding-bottom-9` as far as I'm aware. It's not big enough.

**Comment by josephhull676** [comment]
> shouldnt the word million sit alongside the number?
> 
> Screenshot 2021-11-29 at 18 37 51

Not according to the designs. I guess I'll ask Ife what he thinks!

**Review by alexbowen — APPROVED** [review]

**Pull request merged**